src/commands/utility/holidays.js

```diff
@@ -0,0 +1,161 @@
+const { Command, CommandError, SwitchbladeEmbed } = require('../../')
+const moment = require('moment')
+const fetch = require('node-fetch')
+
+const countries = {
```

Comment:
add country names to locales so it can be translated (countries.json?)

Reply:
Isn't there an NPM module with all country names + translations?

cc @davipatury

Reply:
or just get a data from somewhere

Reply:
I found this package https://github.com/michaelwittig/node-i18n-iso-countries, but how can I know to which language translate?

Reply:
update: solved with countries.json